src/ServiceContainer/SymfonyExtension.php

```diff
@@ -176,12 +178,25 @@ private function loadMinkParameters(ContainerBuilder $container): void
     private function loadBootstrap(?string $bootstrap): void
     {
         if ($bootstrap === null) {
+            $this->loadEnv();
+
             return;
         }
 
         require_once $bootstrap;
     }
 
+    private function loadEnv()
+    {
+        $env = getenv('APP_ENV');
+        try {
+            (new Dotenv())->bootEnv(basename(dirname(__DIR__)).'/../.env', $env);
+        }
+        catch (PathException $exception) {
+            return null;
```

Comment:
What is the point here to return null?

Reply:
I think it was to do something like this if another method of loading variables came up :

```
 if ($bootstrap === null) {
         if( $this->loadEnv() === nul))
            .....
            return;
        }
```

Should we throw an exception if the variables could not be loaded ?

Reply:
Here just to do return, instead of return null.